### DISCUSSION TITLE
Supabase equivalent of firebase Hosting?

### DISCUSSION THREAD

**Discussion by gbersac** [supabase/supabase#8768] [Questions] [answered]
If I create a react single page app, where do I host it so that I can deliver it to our users? 
How do I set it up so that my domain url will redirect to that single page app?
Do I have to user a third party or is it included in the supabase plan?

**Comment by GaryAustin1** [comment, marked as answer]
You would need to use a 3rd party like Vercel, Netlify, Cloudflare.

 This is from edge functions blog post, but mentions hosting plans.

 _Can I host static sites_

_Even though Deno supports [rendering jsx and tsx files](https://deno.land/manual/jsx_dom/jsx), we don't plan on becoming a website hosting provider with Edge Functions. For now, Edge Functions can only be invoked via the POST method. Offering secure and fast web hosting with a good developer experience is a hard problem to solve and there are [other](https://www.netlify.com/) [fantastic](https://vercel.com/) [providers](https://pages.cloudflare.com/) which do exactly that._

_From our experience launching Supabase Storage last year, becoming a hosting provider is not something you want to take lightly. Malicious folks will upload their phishing sites whenever they get a chance and this can quickly spiral out of control, causing you to land on the denylist in the [Safe Browsing dataset](https://safebrowsing.google.com/). Even though we removed the project within minutes after being notified, various ISPs had already cached the information from the Safe browsing dataset and were “helping” their users by restricting access to the Supabase domain. This isn't something we're planning to repeat. But as always, we'd love to hear your use cases and feature requests, we're just one [GitHub discussion](https://github.com/supabase/supabase/discussions) away!_

https://supabase.com/blog/supabase-edge-functions
